Update/kake_076.txt

```diff
@@ -76,7 +76,7 @@ void main()
 //r私の『心』は数ヶ月前に処方された薬を飲むようになってから、ずいぶんと強くなっていた。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "私の『心』は数ヶ月前に処方された薬を飲むようになってから、ずいぶんと強くなっていた。",
-			NULL, "My <i>heart</i> has gotten significantly stronger since I started taking this prescription a few months ago.", Line_Normal);
+			NULL, "My <i>mind</i> has gotten significantly stronger since I started taking this prescription a few months ago.", Line_Normal);
```

Comment:
```suggestion
			NULL, "My <i>heart</i> has gotten significantly stronger since I started taking this prescription a few months ago.", Line_Normal);
```